onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2758,16 +2758,16 @@ static_assert(offsetof(OrtApi, KernelContext_GetResource) / sizeof(void*) == 265
 static_assert(offsetof(OrtApi, SetUserLoggingFunction) / sizeof(void*) == 266, "Size of version 17 API cannot change");
 
 // So that nobody forgets to finish an API version, this check will serve as a reminder:
-static_assert(std::string_view(ORT_VERSION) == "1.18.0",
+static_assert(std::string_view(ORT_VERSION) == "1.19.0",
               "ORT_Version change detected, please follow below steps to ensure OrtApi is updated properly");
 // 1. Update the hardcoded version string in above static_assert to silence it
-// 2. If there were any APIs added to ort_api_1_to_18 above:
+// 2. If there were any APIs added to ort_api_1_to_19 above:
 //    a. Add the 'End of version #' markers (pattern above should be obvious)
 //    b. Add a static_assert in the directly above list of version sizes to ensure nobody adds any more functions to the just shipped API version
```

Comment:
do these steps need to be done?

Reply:
good catch - yes, please update/add static_assert for the newly added/just shipped APIs in this version.